docs/Installation.md

```diff
@@ -228,22 +228,10 @@ See the [post-installation] section for post-installation matters.
 > **Note:** Zonemaster::LDNS and Zonemaster::Engine are not listed here as they
 > are dealt with in the [prerequisites](#prerequisites) section.
 
-Install dependencies available from binary packages:
-
-```sh
-sudo apt-get install libclass-method-modifiers-perl libconfig-inifiles-perl libdata-dump-perl libdbd-sqlite3-perl libdbi-perl libfile-sharedir-perl libfile-slurp-perl libhtml-parser-perl libintl-perl libio-captureoutput-perl libjson-pp-perl libmoose-perl libplack-perl librole-tiny-perl librouter-simple-perl libstring-shellquote-perl libtest-requires-perl libtest-warn-perl libtext-microtemplate-perl libtie-simple-perl starman
-```
-
-Install dependencies not available from binary packages:
-
-```sh
-sudo cpan -i Daemon::Control JSON::RPC Net::IP::XS Parallel::ForkManager Plack::Middleware::Debug
-```
-

```

Comment:
We can have two approaches:

1. Install as much of the Perl dependencies as binary packages.
2. Let cpan install all Perl dependencies.

I think we have followed principle 1 so far. Now you have removed installation of Perl dependencies as binary packages. @sandoche2k, can you explain why?

Reply:
AFAIK we had to install the dependencies since we could install the zonemaster module without pre-installing them. When we had to install the dependencies, we had the two approaches that you mentioned above.

Now with CPANM if it is installing by itself the required dependencies, i do not understand why we should pre-install them.

Reply:
From the top of my mind I can see two good reasons why we should choose OS packages whenever we can.
* To be reasonably sure that people don't install newer versions than the ones we tested with during installation testing.
* To make sure Zonemaster works with the packaged versions of libraries for the OSes we support. This is important since we want OS packaging for Zonemaster.

Reply:
>> whenever we can
That is the question i have too. Either we install everything from OS packages or everything from CPAN. 
Until now, we had the installation instructions wherein if there is a module to be installed, then we look at the binary packages. If we do not get them, then we install them using CPAN. It was adhoc. 
Now by letting CPANM installing them and reducing the burden for the user, we have new requirements. I think we should be coherent on what we want?

Reply:
I think there are pros and cons. By using the CPAN approach the installation instructions will be simpler. If we follow that approach I think that we should take note of the versions of as many libraries as possible to make sure that Makefile.PL is updated with what version that is required. To make it simple for us, require the lowest version that we have found to work.

I guess that the con side is that there is not much help for packaging Zonemaster.

Reply:
While it is possible to lock down the versions of our dependencies in Makefile.PL, I don't think that's the solution we're looking for. We'd have to do specify the versions of all transitive dependencies rather than just the direct dependencies like we do now.